sdk/compiler/damlc/tests/daml-test-files/CCTPMintToken.daml

```diff
@@ -6,9 +6,14 @@
 module CCTPMintToken where
 
 import DA.Assert ((===))
-import DA.Crypto
+import DA.Crypto.Text
 import Daml.Script
 
+type Bytes32Hex = Text
+type UInt32Hex = Text
+type UInt64Hex = Text
+type UInt256Hex = Text
```

Comment:
Work on the `Bytes` primitive data type suggested that its better to place these types in user land - that way they maintain them and leave the language free to evolve primitive versions of these types at a future point in time